### PR TITLE
💄 Revise the styles of Pocket Garden

### PR DESCRIPTION
**Pull request by MaxMuecket**
- Work on responsive design.

- Fix a bug

- Remove code or files.

- Improve structure

- Delete unnecessary code

👀 https://pocket-garde-pocket-gar-zrawjz.herokuapp.com/ 👀

### CODE REVIEW

**Review by SGierszewski — COMMENTED** [review]
Good job! 👍

File: src/app/pages/GardenPage/GardenPage.module.css [hunk 1]
```diff
@@ -21,15 +21,22 @@
...
   .container {
-    background-image: url('../../assets/cover_iPhone_6.png');
     overflow: hidden;
   }
 }
 
+@media (min-height: 668px) {
+  .main {
+    display: grid;
+    grid-template-columns: 4% 43% 3% 48%;
```
Comment (J-Bannach) [inline src/app/pages/GardenPage/GardenPage.module.css:35]: Looks like a complicated grid 😉

File: src/server/routes.ts [hunk 2]
```diff
@@ -22,10 +22,6 @@ router.post('/plants', async (req, res, next) => {
 router.get('/plants', async (_req, res, next) => {
   try {
     const plants = await findPlants();
-    if (!plants) {
```
Comment (J-Bannach) [inline src/server/routes.ts:25]: Are you sure you don't need this? 🙃

**Review by J-Bannach — APPROVED** [review]
Lgtm! 🎉Crazy grids...🤯

File: src/server/routes.ts (hunk 2 above)
Comment (MaxMuecket) [reply src/server/routes.ts:25]: Yes, because it will be always given me an empty array back and never nothing.

**Pull request merged**